CHANGELOG.md

```diff
@@ -6,6 +6,14 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [7.1.0]
+### Uncategorized
```

Comment:
Could you update the changelog? These should all be in a category, and irrelevant entries removed (I think the latest PR is the only one with relevant changes)